stores/Todo.store.ts

```diff
@@ -1 +1,49 @@
-class TodoStore {}
+import { createContext } from "react";
+import { action, makeAutoObservable } from "mobx";
+
+import { loadFromStorage } from "../gateways/Todo.gateway";
+
+export type Todo = {
+  text: string;
+};
+
+export class TodoStore {
+  private _todos: Todo[] = [];
+  private _isLoaded: boolean = false;
+
+  constructor() {
+    this.initialiseTodos();
```

Comment:
I would suggest first call the 'makeAutoObservable' and then call the rest of initialization

Reply:
How come? Does it make a difference?